provider/build.gradle

```diff
@@ -6,22 +6,20 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 34
-        versionCode = Long.valueOf(VERSION_CODE)
-        versionName = VERSION_NAME
```

Comment:
I'm guessing we don't need to define his within the new version of Gradle?

Reply:
It's just moved for clarity and consistency.

When you look further down in `buildTypes`, the `buildConfigField` lines don't refer to `defaultConfig...` anymore but reference the variables directly.